### PR TITLE
feat: [CDS-44153]: added updatereleaserepo step (#11589)

### PR DESCRIPTION
**Pull request by sahithikolichala**
* feat: [CDS-44153]: added updatereleaserepo step

* feat: [CDS-44153]: added updatereleaserepo step

* fix: [CDS-44153]: made all ux changes

* fix: [CDS-44153]: added coverage

* fix: [CDS-44153]: added tests and coverage

* fix: [CDS-44153]: added coverage

* fix: [CDS-44153]: added coverage

* fix: [CDS-44153]: added coverage

* fix: [CDS-44153]: added coverage

* fix: [CDS-44153]: added coverage

* fix: [CDS-44153]: added coverage

* fix: [CDS-44153]: removed console

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

#### [Contributor license agreement](https://github.com/harness/harness-core-ui/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

[CDS-44153]: https://harness.atlassian.net/browse/CDS-44153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CDS-44153]: https://harness.atlassian.net/browse/CDS-44153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CDS-44153]: https://harness.atlassian.net/browse/CDS-44153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CDS-44153]: https://harness.atlassian.net/browse/CDS-44153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CDS-44153]: https://harness.atlassian.net/browse/CDS-44153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CDS-44153]: https://harness.atlassian.net/browse/CDS-44153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CDS-44153]: https://harness.atlassian.net/browse/CDS-44153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CDS-44153]: https://harness.atlassian.net/browse/CDS-44153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CDS-44153]: https://harness.atlassian.net/browse/CDS-44153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CDS-44153]: https://harness.atlassian.net/browse/CDS-44153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CDS-44153]: https://harness.atlassian.net/browse/CDS-44153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CDS-44153]: https://harness.atlassian.net/browse/CDS-44153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 112 Issues 
* Code issue 112 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11669)

File: src/modules/75-cd/components/UpdateReleaseRepo/UpdateReleaseRepo.tsx [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+function UpdateReleaseRepoWidget(props: UpdateReleaseRepoProps, formikRef: StepFormikFowardRef): React.ReactElement {
+  const { initialValues, onUpdate, isNewStep, readonly, allowableTypes, onChange, stepViewType } = props
+  const { getString } = useStrings()
+
+  return (
+    <>
+      <Formik<UpdateReleaseRepoFormData>
+        /* istanbul ignore next */
+        onSubmit={submit => {
+          /* istanbul ignore next */
```
Comment (mtran7) [inline src/modules/75-cd/components/UpdateReleaseRepo/UpdateReleaseRepo.tsx:78]: should this be ignored? Perhaps ignore once coverage set somewhere else like Cypress so we don't miss coverage.

**Review by mtran7 — APPROVED** [review]

**Review by kapilkumawat86 — DISMISSED** [review]

**Review by tan-nhu — DISMISSED** [review]

**Review by cjlee01 — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "nextgenui",
   "description": "Harness Inc",
-  "version": "0.322.7",
+  "version": "0.322.8",
```
Comment (GeorgiosP) [inline package.json:4]: @anshul-harness version has been updated

**Comment by GeorgiosP** [comment]
@cjlee01 @mtran7 @tan-nhu @kapilkumawat86 Please stamp again I had to rebase changes and update package.json version to `0.322.8` Thanks.

**Pull request merged**